src/operator/tensor/elemwise_unary_op_basic.cc

```diff
@@ -106,6 +106,35 @@ The storage type of ``sigmoid`` output is always dense
 
 MXNET_OPERATOR_REGISTER_BINARY_WITH_SPARSE_CPU(_backward_sigmoid,
                                                unary_bwd<mshadow_op::sigmoid_grad>);
+
+DMLC_REGISTER_PARAMETER(HardSigmoidParam);
+MXNET_OPERATOR_REGISTER_UNARY(hard_sigmoid)
+.describe(R"code(Computes hard sigmoid of x element-wise.
+
```

Comment:
You could add an example here.

Reply:
It's unnecessary for this kind of function, please refer to "sigmoid" function above.